mep/books/urls.py

```diff
@@ -11,4 +11,6 @@
         name='work-autocomplete'),
     url(r'^books/(?P<slug>[\w-]+)/$', views.WorkDetail.as_view(),
         name='book-detail'),
+    url(r'^books/(?P<slug>[\w-]+)/circulation$', views.WorkCirculation.as_view(),
```

Comment:
Should have a trailing slash (django convention).

Reply:
oops, thanks for flagging!